eng/pipelines/autorest_checks.yml

```diff
@@ -25,6 +25,7 @@ jobs:
     displayName: 'Run AutoRest'
 
     pool:
+      name: azsdk-pool-mms-ubuntu-2004-general
```

Comment:
At some point it would be good to have these other pipelines include image.yml and just reference those shared variables for the image so we can make updating easier.